### ISSUE TITLE
SPAM pugs never initiate captain picking when filled

### ISSUE THREAD

**Issue by fsk10** [dzejkob-b/discord_irc_pugbot#77]
When a SPAM pug (in #spam channel) is filled it never picks random captains or allow for anyone to become captain.

![spam-pug-bug](https://user-images.githubusercontent.com/65980437/89132624-db63c600-d515-11ea-89e7-3113eca5d4c4.png)

**Comment by oreleraki** [comment]
Mate, I suspect it is the new "playMultiPugs" option, so it will be better if @dzejkob-b  will have a look at it before me.
Also if you can disable that option ("playMultiPugs") i think it will work.

I think this is the line
https://github.com/dzejkob-b/discord_irc_pugbot/blob/14d007a1ffd98b3c1c59f6380e179021e4430830/lib/channel_action.js#L129

But i didn't see any queuing system on the "operRef\botRef" class.

**Comment by dzejkob-b** [comment]
That cannot be the cause. The behavior is weird. I added few things which maybe helps + added principle to store logs after bot reboots (so far logs have been rewriten so cannot investigate more). If it happens again, try .captainforce command (needs admin permission) + maybe ill see something in the log

**Comment by oreleraki** [comment]
I don't think it is the reason for the issue, i just think because `playMultiPugs` is ON, then the `logicLoop` in `botRef` should have some kind of queuing for "initiating captains pickups" when a previous pickup is over.
But i don't totally dig into the code, so i'm no as sure.
